includes/configs/scripts.php

```diff
@@ -11,6 +11,20 @@
 defined( 'ABSPATH' ) || exit;
 
 return [
+	'iris'                  => [
+		'handle' => 'iris',
+		'src'    => admin_url( 'js/iris.min.js' ),
+		'deps'   => [ 'wp-i18n', 'jquery-ui-draggable', 'jquery-ui-slider', 'jquery-touch-punch' ],
+		'scope'  => [ 'frontend' ],
+	],
+
+	'color_picker'          => [
```

Comment:
Make sure that this is the minimum required scripts, is "wp-color-picker" required? Is it a script based on Iris or a separate one, is loading Iris required?

Reply:
Iris and wp-color-picker scripts are needed to stable work